tests/manager/manager-executeCommand-001.phpt

```diff
@@ -37,62 +37,36 @@ object(MongoDB\Driver\Command)#%d (1) {
 }
 bool(true)
 object(MongoDB\Driver\Cursor)#%d (%d) {
-  ["cursor"]=>
-  array(%d) {
-    ["stamp"]=>
-    int(0)
-    ["is_command"]=>
-    bool(true)
-    ["sent"]=>
-    bool(true)
-    ["done"]=>
-    bool(false)
-    ["end_of_event"]=>
-    bool(false)
-    ["in_exhaust"]=>
-    bool(false)
-    ["has_fields"]=>
-    bool(false)
-    ["query"]=>
-    object(stdClass)#%d (1) {
+  ["database"]=>
+  string(6) "phongo"
+  ["collection"]=>
+  NULL
+  ["query"]=>
+  NULL
+  ["command"]=>
+  object(MongoDB\Driver\Command)#%d (%d) {
+    ["command"]=>
+    object(stdClass)#%d (%d) {
       ["ping"]=>
       int(1)
     }
-    ["fields"]=>
-    object(stdClass)#%d (0) {
-    }
-    ["read_preference"]=>
-    array(2) {
-      ["mode"]=>
-      int(1)
-      ["tags"]=>
-      array(0) {
-      }
-    }
-    ["flags"]=>
-    int(0)
-    ["skip"]=>
-    int(0)
-    ["limit"]=>
-    int(1)
-    ["count"]=>
-    int(1)
-    ["batch_size"]=>
-    int(0)
-    ["ns"]=>
-    string(11) "phongo.$cmd"
-    ["current_doc"]=>
-    object(stdClass)#%d (1) {
-      ["ok"]=>
-      float(1)
-    }
   }
-  ["server_id"]=>
-  int(1)
+  ["readPreference"]=>
+  NULL
+  ["isDead"]=>
+  bool(true)
+  ["currentIndex"]=>
+  int(0)
+  ["currentDocument"]=>
+  NULL
+  ["server"]=>
+  object(MongoDB\Driver\Server)#%d (%d) {
+    %a
```

Comment:
The goal of these tests was to assert on `server_id`, so I think we're OK with ignoring the Server's debug output here. The alternative is including it and ignoring `last_is_master` with `%a`. In tests where we do want to check that a cursor operated on a particular server, we do so with `Cursor::getServer()`.